kumascript/macros/ReadOnlyInline.ejs

```diff
@@ -1,34 +1,74 @@
 <%
-var str = "Read only ";
-var title = "This value may not be changed.";
-
-switch(env.locale) {
-    case "de":
-        str = "Schreibgeschützt ";
-        title = "Dieser Wert kann nicht verändert werden.";
-        break;
-    case "fr":
-        str = "Lecture seule ";
-        title = "Cette valeur ne peut pas être changée.";
-        break;
-    case "ja":
-        str = "読取専用 ";
-        break;
-    case "pt-BR":
-        str = "Somente leitura ";
-        title = "Este valor não pode ser alterado.";
-        break;
-    case "ru":
-        str = "Только для чтения ";
-        title = "Это значение изменять нельзя.";
-        break;
-    case "zh-CN":
-        str = "只读 ";
-        title = "该属性的值无法更改";
-        break;
-    default:
-        break;
-}
+var str = mdn.localString({
+    "ca": "Read only ",
+    "cs": "Read only ",
+    "de": "Schreibgeschützt ",
+    "el": "Read only ",
+    "es": "Read only ",
+    "fa": "Read only ",
+    "fi": "Read only ",
+    "fr": "Lecture seule ",
+    "he": "Read only ",
+    "hr": "Read only ",
+    "hu": "Read only ",
+    "id": "Read only ",
+    "it": "Read only ",
+    "ja": "読取専用 ",
+    "ka": "Read only ",
+    "ko": "Read only ",
+    "ms": "Read only ",
+    "nl": "Read only ",
+    "pl": "Read only ",
+    "ro": "Read only ",
+    "ru": "Только для чтения ",
+    "sq": "Read only ",
+    "th": "Read only ",
+    "tr": "Read only ",
+    "vi": "Read only ",
+    "bn-BD": "Read only ",
+    "fy-NL": "Read only ",
+    "ga-IE": "Read only ",
+    "pt-BR": "Somente leitura ",
+    "pt-PT": "Read only ",
+    "zh-CN": "只读 ",
+    "zh-TW": "Read only ",
+    "en-US": "Read only "
```

Comment:
@KhooHaoYit while you are in here, we can do some cleanup. The way `localString` works is that, if it cannot find an entry for the current locale, it will fall back to `en-US` and only return `undefined` if there is no `en-US` entry. That means that, for all the locale strings here, we only need to keep those that are actually translated.

```suggestion
    "de": "Schreibgeschützt ",
    "fr": "Lecture seule ",
    "ja": "読取専用 ",
    "ru": "Только для чтения ",
    "pt-BR": "Somente leitura ",
    "zh-CN": "只读 ",
    "en-US": "Read only "
```

Reply:
Alright, thanks for your suggestions, will change them to your suggested code.